requirements-dev.in

```diff
@@ -13,4 +13,4 @@ requests-mock==1.6.0
 watchdog==0.9.0
 mypy==0.782
 
-git+https://github.com/alphagov/digitalmarketplace-test-utils.git@2.8.0#egg=digitalmarketplace-test-utils==2.8.0
+digitalmarketplace-test-utils
```

Comment:
Should we be specifying the version here, like the other dependencies?

Reply:
No, in fact there is no need to be specifying the versions for the other dependencies here... I will remove that

Reply:
Thanks for pointing that out